### DISCUSSION TITLE
Doesn't work well with some game

### DISCUSSION THREAD

**Discussion by bimash** [nefarius/DsHidMini#121] [Support]
I've tried the software with Persona 4 Golden, for some reason the game goes lag like every 4th lag the controller suddenly does not work. It always happen even though I reconnect the controller, it never happen with any other game like Hades, just Persona 4 Golden.

Probably there's a workaround for it or something about the game that mess with DsHidMini?

**Comment by Kanuan** [comment]
The new XInput mode is in Beta and has some [known issues](https://github.com/ViGEm/DsHidMini/discussions/114).

Try to use DsHidMini in DS4Windows mode instead to check if it works as normal there. [FOLLOW THE GUIDE](https://vigem.org/projects/DsHidMini/DS4-Mode-User-Guide/)

I will convert this from an issue to a discussion topic for now
